udata/templates/dataset/card.html

```diff
@@ -17,75 +17,51 @@
-            <li>
-                <a class="btn btn-xs" v-tooltip
-                    title="{{ _('Territorial coverage') }}">
-                    <span class="fa fa-map-marker fa-fw"></span>
-                    {{ dataset.spatial.top_label }}
-                </a>
+            <li v-tooltip title="{{ _('Number of resources') }}">
+                <span class="{{ ficon('fa-cubes') }} fa-fw"></span>
+                {{ dataset.resources | length  }}
             </li>
-            {% endif %}
             {% if dataset.metrics %}
-            <li>
-                <a class="btn btn-xs" v-tooltip
-                    title="{{ _('Reuses') }}">
-                    <span class="{{ ficon('fa-retweet') }} fa-fw"></span>
-                    {{ dataset.metrics.reuses or 0 }}
-                </a>
+            <li v-tooltip title="{{ _('Reuses') }}">
+                <span class="{{ ficon('fa-retweet') }} fa-fw"></span>
+                {{ dataset.metrics.reuses or 0 }}
             </li>
-            <li>
-                <a class="btn btn-xs" v-tooltip
-                    title="{{ _('Stars') }}">
-                    <span class="{{ ficon('fa-star') }} fa-fw"></span>
-                    {{ dataset.metrics.followers or 0 }}
-                </a>
+            <li v-tooltip title="{{ _('Stars') }}">
+                <span class="{{ ficon('fa-star') }} fa-fw"></span>
+                {{ dataset.metrics.followers or 0 }}
             </li>
             {% endif %}
-        </ul>
 
-    </footer>
-
-
-    <a class="rollover fade in" href="{{url_for('datasets.show', dataset=dataset)}}"
-        title="{{ dataset.title }}">
-        {{ dataset.description|mdstrip(180) }}
-    </a>
-    <footer class="rollover fade in">
-        <ul>
             {# Temporal coverage #}
             {% if dataset.temporal_coverage %}
-            <li>
-                <a class="btn btn-xs" v-tooltip title="{{ _('Temporal coverage') }}">
-                    <span class="fa fa-calendar fa-fw"></span>
-                    {{ dataset.temporal_coverage|daterange }}
-                </a>
+            <li v-tooltip title="{{ _('Temporal coverage') }}">
+                <span class="fa fa-calendar fa-fw"></span>
+                {{ dataset.temporal_coverage|daterange }}
             </li>
             {% endif %}
 
             {# Territorial coverage #}
-            {% if dataset.spatial and dataset.spatial.granularity %}
-            <li>
-                <a class="btn btn-xs" v-tooltip title="{{ _('Territorial coverage granularity') }}">
-                    <span class="fa fa-bullseye fa-fw"></span>
-                    {{ dataset.spatial.granularity_label }}
-                </a>
+            {% if dataset.spatial and dataset.spatial.territories %}
+            <li v-tooltip title="{{ _('Territorial coverage') }}">
+                <span class="fa fa-map-marker fa-fw"></span>
+                {{ dataset.spatial.top_label }}
```

Comment:
I could not find any place where this is working. It should display "France" on a bunch of datasets on the HP if I'm not mistaken.

Reply:
blind guess: this is a messed up by some parallel workaround on "territories". I have put back what have been used as a condition to show top zone or not.